<article class="chapter">  <h1 class="module-name">Class ConflictError</h1>  <table> <tr> <th>Extends:</th> <td><a href="adaptererror">AdapterError</a></td> </tr>  <tr> <th>Defined in:</th> <td> <a href="https://github.com/emberjs/data/tree/v3.25.0/packages/-ember-data/../adapter/addon/error.js#L313" target="_blank" rel="noopener noreferrer">../adapter/addon/error.js:313</a> </td> </tr> <tr> <th>Module:</th> <td><a href="../modules/@ember-data/adapter">@ember-data/adapter</a></td> </tr>  </table>  
<p>A <code>ConflictError</code> equates to a HTTP <code>409 Conflict</code> response status. It is used by an adapter to indicate that the request could not be processed because of a conflict in the request. An example scenario would be when creating a record with a client-generated ID but that ID is already known to the external API.</p>  </article><div class="_attribution">
  <p class="_attribution-p">
    <a href="https://api.emberjs.com/ember-data/3.25/classes/ConflictError" class="_attribution-link" target="_blank">https://api.emberjs.com/ember-data/3.25/classes/ConflictError</a>
  </p>
</div>
